content/applications/inventory_and_mrp/manufacturing/quality_control/instructions_check.rst

```diff
@@ -0,0 +1,85 @@
+==========================
+Instructions quality check
+==========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+In Odoo *Quality*, an Instructions check is one of the quality check types that can be selected when
+creating a new quality check or Quality Control Point (QCP). Instructions checks consist of a text
+entry field that allows the creator to provide instructions for how to complete the check.
+
+There are two distinct ways that quality checks can be created. A single check can be manually
+created. Alternatively, a |QCP| can be configured that will automatically create checks at a
+predetermined interval.
+
+This documentation will only detail the configuration options and processing steps that are unique
+to Instructions quality checks. For a full overview of all the configuration options available when
+creating a single check or a |QCP|, see the documentation on :ref:`Quality checks
+<manufacturing/quality_control/quality-checks>` and :ref:`Quality Control Points
+<manufacturing/quality_control/quality-control-points>`.
+
+Create an Instructions quality check
+====================================
```

Comment:
Would this heading still make sense? Totally up to you whether you accept this suggestion or not (:

```suggestion
Create Instructions
===================
```
I just like to keep heading titles as short as possible so it looks clean on the navigation bar here:
![image](https://github.com/odoo/documentation/assets/23611711/4f6c4fcb-fc0f-4961-87c0-e556d648b546)